migrations/cockroach/V1.60__zitadel_schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE zitadel;
+GRANT SELECT, INSERT, UPDATE, DELETE ON DATABASE zitadel TO queries;
+use zitadel;
+
+CREATE SCHEMA projections AUTHORIZATION queries;
+
+CREATE TABLE projections.locks (
+    locker_id TEXT,
+    locked_until TIMESTAMPTZ(3),
+    view_name TEXT,
+
+    PRIMARY KEY (view_name)
+);
+
+CREATE TABLE projections.current_sequences (
+    view_name TEXT,
+    aggregate_type TEXT,
+    current_sequence BIGINT,
+    timestamp TIMESTAMPTZ,
+
+    PRIMARY KEY (view_name, aggregate_type)
+);
+
+CREATE TABLE projections.failed_events (
+    projection_name TEXT,
+    failed_sequence BIGINT,
+    failure_count SMALLINT,
+    error TEXT,
+
+    PRIMARY KEY (projection_name, failed_sequence)
+);
+
+CREATE TABLE projections.orgs (
```

Comment:
wasn't it the purpose to generate the projection tables on the fly

Reply:
It was and still is. I prefer to implement this as soon as all handlers are migrated. It will not impact the current implementation of eventstore but a lot in the business logic as we need an API, create events and handlers for this business request.